canonical-websites/www.ubuntu.com
# remove desktop section scss

## Done

* created two new pattern file for the photo strip and gaming strip on the /desktop features page
* removed the _v1_desktop page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/desktop/features)
- see that the /desktop/feature page
